src/main/java/com/sap/oss/phosphor/fosstars/data/github/CodeOfConductGuidelineInfo.java

```diff
@@ -169,7 +167,7 @@ protected ValueSet fetchValuesFor(GitHubProject project) throws IOException {
    */
   private Value<Boolean> infoAboutCodeOfConductGuideline(String content) {
     if (requiredContentPatterns.isEmpty()) {
-      return HAS_REQUIRED_TEXT_IN_CODE_OF_CONDUCT_GUIDELINE.value(true);
+      return HAS_REQUIRED_TEXT_IN_CODE_OF_CONDUCT_GUIDELINE.value(false);
```

Comment:
Again I wanted an explanation as to why?

- If there are no patterns to search and if the file is found. What does this mean?
- If there is a pattern and if you find the file. Does it mean it is the best?
- If there are patterns to search, but not found in the found file. Does it mean its good?
- If there are patterns to search, but no file found. Does it mean the worst?

Reply:
- ´ requiredContentPatterns ´ are the patterns we want to check in the file. 
If the patterns are empty that means there are not patterns to be found, which means the file (if present) is perfect and we can move on so - the ´HAS_REQUIRED_TEXT´ is always ´true´ as there is nothing to search for. 
- and if ´ requiredContentPatterns ´ is not empty that means we now need to apply regex and check if the pattern exists in the file's text. which we do later. 
- So the original ´return HAS_REQUIRED_TEXT_IN_CODE_OF_CONDUCT_GUIDELINE.value(true);´ was right.
- I will revert the change.